extractor.go

```diff
@@ -69,7 +75,23 @@ func (e *Extractor) Extract(root, rel string) error {
 		if err != nil {
 			return err
 		}
-		return e.processFile(fname, path, sample)
+		if err := e.processFile(fname, path, sample); err != nil {
+			switch {
+			case !strings.Contains(err.Error(), "is a directory"):
```

Comment:
Meh. You should detect it based on the error code. Also, you can filter those even before by looking at `info.Mode()`. It has a flag for symlinks.

Reply:
Done